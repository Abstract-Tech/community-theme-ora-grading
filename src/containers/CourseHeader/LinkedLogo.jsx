import React from 'react';
import { getConfig } from '@edx/frontend-platform';

const LinkedLogo = () => (
  <a
    className="logo"
    href={`${getConfig().LMS_BASE_URL}/dashboard`}
  >
    <img
      className="d-block"
      src={getConfig().LOGO_URL}
      alt={getConfig().SITE_NAME}
    />
  </a>
);

export default LinkedLogo;
