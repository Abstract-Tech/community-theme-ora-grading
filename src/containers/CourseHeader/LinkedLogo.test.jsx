import React from 'react';
import { shallow } from 'enzyme';

import LinkedLogo from './LinkedLogo';

jest.mock('@edx/frontend-platform', () => ({
  getConfig: () => ({
    LMS_BASE_URL: '<getConfig().LMS_BASE_URL>',
    LOGO_URL: '<getConfig().LOGO_URL>',
    SITE_NAME: '<getConfig().SITE_NAME>',
  }),
}));

describe('Header CourseLabel component', () => {
  test('snapshot', () => {
    expect(
      shallow(<LinkedLogo />),
    ).toMatchSnapshot();
  });
});
