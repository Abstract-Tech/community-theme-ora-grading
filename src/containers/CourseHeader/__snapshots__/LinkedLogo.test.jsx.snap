// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header CourseLabel component snapshot 1`] = `
<a
  className="logo"
  href="<getConfig().LMS_BASE_URL>/dashboard"
>
  <img
    alt="<getConfig().SITE_NAME>"
    className="d-block"
    src="<getConfig().LOGO_URL>"
  />
</a>
`;
